erc20-bridge-token/test/BridgeToken.js

```diff
@@ -399,6 +403,109 @@ describe('BridgeToken', () => {
       ).to.be.equal("0");
     });
 
+    it("Test multiple tokens", async function() {
+      const amountToTransfer = amountToLock;
+      const whitelistTokens = [
+        "near-token1.near",
+        "near-token2.near",
+        "near-token3.near",
+        "near-token4.near",
+      ];
+      const blacklistTokens = [
+        "near-token5.near",
+        "near-token6.near",
+        "near-token7.near",
+        "near-token8.near",
+      ];
+
+      const tokensInfo = [];
+      for (token of whitelistTokens) {
+        tokensInfo.push(await createToken(token));
+        await deposit(token, amountToTransfer, user.address);
+        await BridgeTokenFactory.setTokenWhitelistMode(token, 2);
+      }
+
+      for (token of blacklistTokens) {
+        await expect(
+          BridgeTokenFactory.connect(user).withdraw(
+            token,
+            amountToTransfer,
+            recipient
+          )
+        ).to.be.revertedWith("ERR_NOT_INITIALIZED_WHITELIST_TOKEN");
+      }
+
+      for (token of whitelistTokens) {
+        await expect(
+          BridgeTokenFactory.connect(user).withdraw(
+            token,
+            amountToTransfer,
+            recipient
+          )
+        )
+          .to.emit(BridgeTokenFactory, "Withdraw")
+          .withArgs(token, user.address, amountToTransfer, recipient);
+      }
+
+      for (tokenInfo of tokensInfo) {
+        expect(
+          (await tokenInfo.token.balanceOf(user.address)).toString()
+        ).to.be.equal("0");
+      }
+    });
+
+    it("Test multiple accounts", async function() {
+      const amountToTransfer = amountToLock;
+      const whitelistTokens = [
+        "near-token1.near",
+        "near-token2.near",
+        "near-token3.near",
+        "near-token4.near",
+      ];
+
+      const signers = await ethers.getSigners();
+      const whitelistAccounts = signers.slice(0, 3);
+      const blacklistAccounts = signers.slice(4, 7);
```

Comment:
Please use lengths of appropriate account lists for indices.

Reply:
cf35ca224be4e69c77bbbb4b295a8a00eb9cd9e7